INSTALL.md

```diff
@@ -0,0 +1,50 @@
+# Metrics deployment instructions
+## Create app
+```bash
+dokku$ dokku apps:create metrics
+dokku$ dokku git:set metrics deploy-branch main
+```
+
+## Configure app
+```bash
+dokku config:set metrics GITHUB_TOKEN'xxx'
+dokku config:set metrics SLACK_SIGNING_SECRET='xxx'
+dokku config:set metrics SLACK_TECH_SUPPORT_CHANNEL_ID='xxx'
+dokku config:set metrics SLACK_TOKEN='xxx'
+dokku config:set metrics TIMESCALEDB_URL='xxx'
+```
+
+## Disable checks
+Dokku performs health checks on apps during deploy by sending requests to port 80.
+This tool isn't a web app does it can't accept requests on a port.
+Disable the checks so deploys can happen.
+```bash
+dokku$ dokku checks:disable metrics
+```
+
+## Set up storage
+This is only needed for backfills which make use of a SQLite db cache.
+```bash
+# the metrics container runs as uid 1000 (metrics)
+# the other dokku containers on dokku3 run as uid 1013 (dokku)
```

Comment:
I think you're right because permissions are mapped down to the UID right…? Before I did the docker-options config this raised an error, since `bloodearnest` doesn't have permission to write to these directories, which I think is probably ok?